card/db/migrate_core_cards/20141204061304_watchers_to_following.rb

```diff
@@ -22,7 +22,7 @@ def up
     
       follower_hash.each do |user, items|
         if card=Card.fetch(user) and card.account
-          following = card.fetch :trait=>"following", :new=>{}
+          following = Card.fetch("#{card.name}+*following", :new=>{:type=>'pointer'})
```

Comment:
The trait fetch with the type should work the same, right?  The new version will be wrong if the *following card is renamed.  Likewise on the :type param, should be :typecode or :type_id forms, I think.